src/smart-components/group/group-table-helpers.js

```diff
@@ -40,44 +42,116 @@ DefaultPlatformPopover.propTypes = {
+                {name}
+              </AppLink>
+            ) : (
+              name
+            ),
+          },
+          { title: roleCount, props: { isOpen: expanded[uuid] === 2 } },
+          { title: principalCount, props: { isOpen: expanded[uuid] === 3 } },
+          { title: <DateFormat date={modified} type={getDateFormat(modified)} /> },
+        ],
+      },
+      {
+        uuid: `${uuid}-roles`,
+        parent: 3 * i,
+        compoundParent: 1,
+        fullWidth: true,
+        noPadding: true,
+        cells: [
+          {
+            props: { colSpan: 7, className: 'pf-m-no-padding' },
+            title:
+              roleCount.length > 0 ? (
+                <Table
+                  ouiaId="roles-in-group-nested-table"
+                  aria-label="Simple Table"
+                  variant={TableVariant.compact}
+                  cells={[
+                    intl.formatMessage(messages.roleName),
+                    intl.formatMessage(messages.description),
+                    intl.formatMessage(messages.permissions),
+                    intl.formatMessage(messages.lastModified),
+                  ]}
+                  rows={expandTableDetails.map((roles) => {
+                    const [roleName, description, permissions] = roles.permission.split(':');
+                    return {
+                      cells: [
+                        roleName,
+                        description,
+                        permissions,
+                        <Fragment key={`${uuid}-modified`}>
+                          <DateFormat date={modified} type={getDateFormat(modified)} />
+                        </Fragment>,
+                      ],
+                    };
+                  })}
+                >
+                  <TableHeader />
+                  <TableBody />
+                </Table>
+              ) : (
+                <Text className="pf-u-mx-lg pf-u-my-sm">{intl.formatMessage(messages.noGroupRoles)}</Text>
+              ),
+          },
+        ],
+      },
+      {
+        uuid: `${uuid}-members`,
+        parent: 3 * i,
+        compoundParent: 2,
+        fullwidth: true,
+        noPadding: true,
+        cells: [
+          {
+            props: { colSpan: 7, className: 'pf-m-no-padding' },
+            title:
+              principalCount > 0 ? (
+                <Table
+                  aria-label="Simple Table"
+                  ouiaId="members-in-group-nested-table"
+                  variant={TableVariant.compact}
+                  cells={[
+                    intl.formatMessage(messages.orgAdmin),
+                    intl.formatMessage(messages.username),
+                    intl.formatMessage(messages.email),
+                    intl.formatMessage(messages.firstName),
+                    intl.formatMessage(messages.lastName),
+                    intl.formatMessage(messages.status),
+                  ]}
+                  rows={expandTableDetails.map((members) => ({
```

Comment:
```suggestion
                  rows={members.map((member) => ({
```
add at least an empty array, without it, there will be still an error